travis/Dockerfile-travis-web

```diff
@@ -13,5 +13,6 @@ WORKDIR /src
 COPY . /src
 
 RUN chown -R mitodl:mitodl /src
+RUN mkdir /var/media && chown -R mitodl:mitodl /var/media
```

Comment:
This dockerfile inherits from an image which is build from the root level dockerfile, so I don't think we really need to strive for this being the same. The reason for this dockerfile is just to handle the case where a branch introduces a dependency change, we want to make sure we run a reinstall of the dependencies.